src/test/java/hellocucumber/steps/ResultVerificationStep.java

```diff
@@ -0,0 +1,18 @@
+package hellocucumber.steps;
+
+import hellocucumber.TestContext;
+import io.cucumber.java.en.Then;
+import org.testng.Assert;
+
+public class ResultVerificationStep {
+    TestContext testContext;
+
+    public ResultVerificationStep(TestContext context) {
+        this.testContext = context;
+    }
+
+    @Then("the result is {string}")
+    public void the_result_is(String string) {
+        Assert.assertEquals(testContext.getActualResult(), string, "Result is incorrect");
```

Comment:
saying that result is incorrect is not enough and will need debugging to see actual values. I prefer to have messages like: 'Expected to have %d, but got %d instead'